### PR TITLE
QG4 check

### PR DESCRIPTION
**Pull request by adkumar1**
Header addition
CPU, Memory addition

### CODE REVIEW

File: .github/workflows/helm-test.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+            - name: Kubernetes KinD Cluster
+              uses: container-tools/kind-action@v1
+        
+            - name: Build image
+              uses: docker/build-push-action@v3
+              with:
+                context: .
+                push: true
+                tags: ghcr.io/catenax-ng/tx-sd-factory/sdfactory:test
```
Comment (almadigabor) [inline .github/workflows/helm-test.yaml:49]: You shouldn't tag and push to the ghcr package manager, but instead to the local docker registry in the kind cluster like in this: `kind-registry:5000/sd-factory:testing`

File: .github/workflows/helm-test.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+              run: |
+                changed=$(ct list-changed --target-branch ${{ github.event.repository.default_branch }})
+                if [[ -n "$changed" ]]; then
+                   echo "::set-output name=changed::true"
+                fi
+            - name: Run chart-testing (lint)
+              run: ct lint --validate-maintainers=false --target-branch ${{ github.event.repository.default_branch }}
+        
+            - name: Run chart-testing (install)
+              run: ct install --charts charts/app-dashboard
```
Comment (almadigabor) [inline .github/workflows/helm-test.yaml:75]: Also the following should be set to find the image pushed to the kind cluster: `ct install --charts charts/SDFactory --helm-extra-set-args "--set=image.tag=testing"`

File: .github/workflows/helm-test.yaml (hunk 1 above)
Comment (adkumar1) [reply .github/workflows/helm-test.yaml]: Done

**Comment by almadigabor** [comment]
You can see now the results of the helm test workflow with the errors that needs to be addressed. Like in the last run: https://github.com/eclipse-tractusx/sd-factory/actions/runs/4991067895/jobs/8937074989?pr=46#step:9:31

File: .github/workflows/helm-test.yaml (hunk 2 above)
Comment (adkumar1) [reply .github/workflows/helm-test.yaml]: Done

**Comment by almadigabor** [comment]
New issues in the next run: https://github.com/eclipse-tractusx/sd-factory/actions/runs/4991168700/jobs/8937297579?pr=46#step:9:31

File: .github/workflows/helm-test.yaml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+              run: |
+                changed=$(ct list-changed --target-branch ${{ github.event.repository.default_branch }})
+                if [[ -n "$changed" ]]; then
+                   echo "::set-output name=changed::true"
+                fi
+            - name: Run chart-testing (lint)
+              run: ct lint --validate-maintainers=false --target-branch ${{ github.event.repository.default_branch }}
+        
+            - name: Run chart-testing (install)
+              run: ct install --charts charts/SDFactory --helm-extra-set-args "--set=image.tag=testing"
```
Comment (almadigabor) [inline .github/workflows/helm-test.yaml:77]: Can you try the following change?

```suggestion
              run: ct install --charts charts/SDFactory --helm-extra-set-args "--set=image.tag=testing --set=image.repository=sd-factory"
```

**Pull request merged**